### PR TITLE
docs: rename spreadsheet-blueprints to demo-blueprints

### PR DESCRIPTION
**Pull request by alvancamp**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Docs update

* **What is the current behavior?** (You can also link to an open issue here)

The demo blueprints are mentioned as spreadsheet blueprints.

* **What is the new behavior (if this is a feature change)?**

The blueprints are now explicitly called demo blueprints instead of spreadsheet blueprints.

* **Other information**:

I also added a few more links to the [Rundown Editor](https://github.com/SuperFlyTV/sofie-automation-rundown-editor). There's also some unrelated style and formatting changes caused by the auto formatter.

### CODE REVIEW

**Comment by baltedewit** [comment]
@nytamin I think documentation-only pull requests that are already relevant should not follow our normal git branching strategy because that could delay updates to documentation significantly. I'll bring this up for internal discussion.

**Comment by jstarpl** [comment]
> @nytamin I think documentation-only pull requests that are already relevant should not follow our normal git branching strategy because that could delay updates to documentation significantly. I'll bring this up for internal discussion.

Shouldn't we essentially treat it as a bugfix, since well, it's fixing what amounts to a bug in documentation?

**Comment by nytamin** [comment]
> > @nytamin I think documentation-only pull requests that are already relevant should not follow our normal git branching strategy because that could delay updates to documentation significantly. I'll bring this up for internal discussion.
> 
> Shouldn't we essentially treat it as a bugfix, since well, it's fixing what amounts to a bug in documentation?

Agreed, let's do that (and also discuss it internally)

File: packages/documentation/docs/getting-started/installation/installing-a-gateway/rundown-or-newsroom-system-connection/installing-sofie-with-google-spreadsheet-support.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ The Spreadsheet Gateway is an application for piping data between Sofie Core and
 
 ### Example Blueprints for Spreadsheet Gateway
 
-To begin with, you will need to install a set of Blueprints that can handle the data being sent from the _Gateway_ to _Core_. Download the ZIP file containing the three files you need from the [Spreadsheet Gateway's GitHub Repository](https://github.com/SuperFlyTV/sofie-blueprints-spreadsheet/releases). It is recommended to choose the newest release but, an older _Core_ version may require a different Blueprint version. The _Rundown page_ will warn you about any issue and display the desired versions. 
+To begin with, you will need to install a set of Blueprints that can handle the data being sent from the _Gateway_ to _Core_. Download the `bundle.json` file containing the blueprints you need from the [Demo Blueprints GitHub Repository](https://github.com/SuperFlyTV/sofie-demo-blueprints/releases). It is recommended to choose the newest release but, an older _Core_ version may require a different Blueprint version. The _Rundown page_ will warn you about any issue and display the desired versions.
```
Comment (baltedewit) [inline packages/documentation/docs/getting-started/installation/installing-a-gateway/rundown-or-newsroom-system-connection/installing-sofie-with-google-spreadsheet-support.md:7]: I thought you found this `bundle.json` doesn't work?

**Review by baltedewit — CHANGES_REQUESTED** [review]

File: packages/documentation/docs/getting-started/installation/installing-a-gateway/rundown-or-newsroom-system-connection/installing-sofie-with-google-spreadsheet-support.md (hunk 1 above)
Comment (alvancamp) [reply packages/documentation/docs/getting-started/installation/installing-a-gateway/rundown-or-newsroom-system-connection/installing-sofie-with-google-spreadsheet-support.md]: Good catch, fixed.

**Pull request merged**